.github/workflows/rust.yml

```
@@ -0,0 +1,48 @@
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        app-dir: [ hello-koma, actix-gcd ]
```

yamllint

[error] too many spaces inside brackets

(brackets)